src/models/Tabs.ts

```diff
@@ -1,7 +1,13 @@
 export interface TabItemProps {
   label: string
   id: string
-  content: string
+  content: {
+    interestFree: string
+    purchaseAmount: string
+    estFee: string
+    minimumMonthlyPayment: string
+    repayBeforeInterest: string
+  }
```

Comment:
This would also tie in to the coupling of the tabs component mentioned further up. The Tabs shouldn't have to be aware of what the content is